pygmt/x2sys.py

```diff
@@ -221,7 +222,8 @@ def x2sys_cross(tracks=None, outfile=None, **kwargs):
     combitable : str
         Only process the pair-combinations found in the file *combitable*
         [Default process all possible combinations among the specified files].
-        The file *combitable* is created by *x2sys_get*'s -L option
+        The file *combitable* is created by :gmt-docs:`x2sys_get's -L option
+        <supplements/x2sys/x2sys_get.html#l>`.
 
     runtimes : bool or str
         Compute and append the processing run-time for each pair to the
```

Comment:
Line 233: "See the MATLAB function split_file4coes.m that lives in the x2sys supplement source code."

Link to https://github.com/GenericMappingTools/gmt/blob/master/src/x2sys/split_file4coes.m

Reply:
I linked to the folder rather than the file to keep it under 80 characters.